WorkspaceServer.Tests/TestProjects/SampleConsole/a.html

```diff
@@ -0,0 +1 @@
+<p>This file should be served</p>
```

Comment:
It would be better to move the test file to the project that actually depends on it. It's unrelated to the scenario for which the `SampleConsole` directory was created, which is to test the `IDirectoryAccessor`.